cypress.json

```diff
@@ -15,7 +15,7 @@
   "watchForFileChanges": true,
   "nodeVersion": "system",
   "env": {
-    "OC_IDP": "kube:admin"
+    "OPTIONS_HUB_OC_IDP": "kube:admin"
```

Comment:
Updating the value name to `OPTIONS_HUB_OC_IDP` to remain more consistent with the other value names.